tests/unit_tests/c_wrappers/res/enkf/test_ensemble_config.py

```diff
@@ -84,5 +84,4 @@ def test_ensemble_config_constructor(setup_case):
                 }
             ],
         },
-        grid=res_config.ensemble_config.grid,
```

Comment:
This change seems to be in the wrong commit

Reply:
A slip up! Thanks! :wrench: